jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants.properties

```
@@ -187,3 +187,4 @@ NavigationConfirmOkButtonText=Ok
 NavigationConfirmMessage=Are you sure you want to go to Sub Process Diagram?
 AlertPopupTitle=Can not navigate to Sub Process Diagram
 AlertPopupMessage=There are multiple instances of the sub-process, please use the Sub Process Instances Panel instead.
+CanNotGetTimerInstancesMessage=A long task is running and Timers cannot be displayed. Check again later.
```

This failure could be caused by other issues so I would recommend living it more generic, something like:
```suggestion
CanNotGetTimerInstancesMessage=Error trying to retrieve process instance timers details. Message: {0}
```

---

ok

---

Done